smk-create/index.js

```diff
@@ -3,6 +3,7 @@ const fs = require( 'fs' )
 const path = require( 'path' )
 const inquirer = require( 'inquirer' )
 const shell = require( 'shelljs' )
+const semverGte = require ( 'semver/functions/gte' )
```

Comment:
True, but if I remove the extra spaces, there's so much other extra whitespace in this file that it would look weird for just this section to be updated. I'll leave it as-is for now, but I did create a tech debt task (MAP-3297) to look into using a linter or similar to find and fix these inconsistencies in syntax throughout the codebase.